content/source/docs/enterprise/private/monitoring.html.md

```diff
@@ -18,4 +18,16 @@ In addition to health-check monitoring, we recommend monitoring standard server
 
 - I/O
 - CPU
-- Disk
\ No newline at end of file
+- Disk
+
+## Internal Monitoring
+
+Beginning in version 201811-1, PTFE includes internal monitoring of critical application metrics using a statsd collector on port 8125. The resulting metrics are included with the [diagnostic bundles provided to HashiCorp Support](./diagnostics.html). If the PTFE instance is already running a collector on this port, PTFE may not start up correctly due to the conflict, and logs will indicate:
```

Comment:
Question! Why would the instance already be running a collector on that port? Would it be from a zombie copy of the normal collector, or something else?

Reply:
Some of our customers run their own stats collectors!

Reply:
Oh huh, yeah, I guess the ptfe instance can run other software without having to cram it into our docker images. I always forget that. 😅